src/components/Header.astro

```diff
@@ -60,7 +60,7 @@ import { SITE_TITLE } from "../consts";
               <a class="text-lg font-bold">About</a>
               <ul class="p-2">
                 <li><a href="/about" class="text-md">About PyCon</a></li>
-                <li><a href="/psf" class="text-md">PyCon PSF</a></li>
+                <li><a href="/the-psf" class="text-md">The PSF</a></li>
```

Comment:
why the link must be "the-psf"

Reply:
the content refers to The PSF US, so I just change it to that